src/OpenColorIO/Config.cpp

```diff
@@ -3812,6 +3944,13 @@ const char * Config::getVirtualDisplayViewColorSpaceName(const char * view) cons
 {
     if (!view) return "";
 
+    // TODO: Remove the following work-around once bug for shared views is fixed.
```

Comment:
Actually, despite what I wrote, you could remove the TODO, I think your fix is sufficient.